ci/clone-and-checkout-pr.py

```diff
@@ -41,10 +41,13 @@
 cmd = "git clone " + str(repo.clone_url) + " repository"
 
 child = pexpect.spawn(cmd)
-child.expect("Username.*")
-child.sendline(username + "\n")
-child.expect("Password.*")
-child.sendline(password + "\n")
+
+if '-dev' in REPO_NAME:
```

Comment:
Is this because for openNetVM-dev, there are already permissions? So someone can only pull if they are in our team? Not an edit, just a question.

Reply:
yep, if you have access to -dev -> it's assumed you can run CI